src/add.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/src/index.scss"/>
+
+    <title>Dicționar Urban</title>
+</head>
+<body>
+    <div class="thebody">
+        <header class="theheader">
+            <div class="wrapper">
+                <div class="logo">
+                    <a href="/src/index.html"><img src="/src/img/logo.svg" class="logo" alt="logo"/></a>
+                </div>
+                <div class="navbar">
+                    <div class="close-nav"><button>X</button></div>
+                    <nav>
+                        <ul>
+                            <li><a href="/src/index.html">Acasă</a></li>
+                            <li><a href="/src/add.html">Adaugă</a></li>
+                            <li><a href="/src/login.html">Autentificare</a></li>
+                            <li><a href="/src/contact.html">Contact</a></li>
+                        </ul>
+                    </nav>
+                </div>
+                <div class="menu-bar">
+                    <button>&#9776;</button>
+                </div>
+                <form id="search-form" method="get" action="">
+                <input class="search-input" type="search" placeholder="Caută...">
+                <button type="submit" class="search-button">
+                    <img src="/src/img/search.svg">
+                </button>
+                </form> 
+            </div>     
+        </header>
+        <div class="container">
+            <div class="content">
+                <div class="add-page">
+                    <h2>ADAUGĂ O EXPRESIE</h2>
+                    <form action="" id="the-form" method="post" class="add-form">
+                        <label for="expression"><b>Expresie</b></label>
+                        <textarea name="expression" placeholder="Introdu expresia" rows="1" cols="20" required></textarea>
```

Comment:
Vezi că în HTML5 `for` se referă la ID-ul elementului, nu la nume.
```suggestion
                        <label for="expression"><b>Expresie</b></label>
                        <textarea id="expression" name="expression" placeholder="Introdu expresia" rows="1" cols="20" required></textarea>
```